### ISSUE TITLE
About the dataset WebVid-CoVR

### ISSUE THREAD

**Issue by zhang252112914** [lucas-ventura/CoVR#34]
Thank you for creating such an excellent benchmark. However, I still have some questions about the dataset. 

Based on the methodology described for creating the dataset, the videos used appear to come from WebVid2M and WebVid10M, but not necessarily all of them, right? If that's the case, could the authors provide a concise version of the video portion used?

Additionally, I noticed that the test set is a subset of WebVid10M. Could the authors provide this subset directly without requiring access to the full WebVid10M?

**Comment by lucas-ventura** [comment]
Hi, thanks for your kind words and for your interest in our dataset!

You're right, the videos in the training set all come from WebVid2M, but we don’t use the entire set. Similarly, the test set videos come from WebVid10M (excluding the 2M subset from WebVid2M), and again, we only use a portion of the full dataset.

You can download exactly the videos we use by following the instructions in the README under **Installation > Download the datasets**. Specifically, you can run:

```bash
python tools/scripts/download_covr.py --split=<train, val or test>
```

If you'd like to see exactly which videos are included in each split, feel free to check out the [`download_covr.py`](https://github.com/lucas-ventura/CoVR/blob/master/tools/scripts/download_covr.py) script, it has links to the train, val, and test lists with only the necessary videos.

Hope this helps! Let me know if you have any further questions.

**Comment by zhang252112914** [comment]
Thank you for your timely response. I truly appreciate your help. I had focused all my attention on the datasets section of Hugging Face and accidentally overlooked the readme on GitHub.😁
